<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attendance</class>
 <widget class="QWidget" name="Attendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOut">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>180</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>考勤查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>部门：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combDepart">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>行政部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工程部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>技术部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>销售部</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>工号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSno">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>全部</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>911</width>
     <height>301</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>工号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>部门</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>上班签到时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>打卡情况</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>下班签到时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>打卡情况</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>工作状态</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnSno">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>单工号查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDepart">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>单部门查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcel">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color:rgb(0, 255, 0);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(0, 170,0);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(0, 255, 0);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>生成Excel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
